.github/workflows/ci.yml

```diff
@@ -30,6 +30,27 @@ jobs:
         with:
           name: ci-artifacts
           path: artifacts-jvm${{ matrix.java }}.zip
+  build-released-native:
+    name: Native build - released Quarkus
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        java: [ 11 ]
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/cache@v1
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+          restore-keys: |
+            ${{ runner.os }}-maven-
+      - name: Install JDK {{ matrix.java }}
+        uses: joschi/setup-jdk@v1.0.0
+        with:
+          java-version: openjdk${{ matrix.java }}
+      - name: Build with Maven
```

Comment:
I don't think it's necessary to have 2 new builds, we could have just native step added to the existing builds.

What's your motivation to have separate native builds? Is it to have separated native vs. jvm ++ more concurrent builds / earlier feedback ...
The downside of this approach is that more instances are needed (not big deal for us) ++ Quarkus master is build twice.

An example without adding new builds:
```diff
diff --git a/.github/workflows/ci.yml b/.github/workflows/ci.yml
index eca0e1a..5c49744 100644
--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -4,7 +4,7 @@ on:
   - pull_request
 jobs:
   build-released:
-    name: JVM build - released Quarkus
+    name: Build - released Quarkus
     runs-on: ubuntu-latest
     strategy:
       matrix:
@@ -23,6 +23,9 @@ jobs:
           java-version: openjdk${{ matrix.java }}
       - name: Build with Maven
         run: mvn -V -B clean test
+      - name: Native Build
+        run: |
+          mvn -V -B clean package -DskipTests -DskipITs -pl 'app-metadata/deployment,app-metadata/runtime,common,http' -Dquarkus.profile=native -Dquarkus.native.container-runtime=docker
       - name: Zip Artifacts
         run: |
           zip -R artifacts-jvm${{ matrix.java }}.zip 'surefire-reports/*'
@@ -31,7 +34,7 @@ jobs:
           name: ci-artifacts
           path: artifacts-jvm${{ matrix.java }}.zip
   build-master:
-    name: JVM build - Quarkus master
+    name: Build - Quarkus master
     runs-on: ubuntu-latest
     strategy:
       matrix:
@@ -52,6 +55,9 @@ jobs:
         run: git clone https://github.com/quarkusio/quarkus.git && cd quarkus && mvn -B clean install -DskipTests -DskipITs -DskipDocs
       - name: Build with Maven
         run: mvn -V -B clean test -Dquarkus-core-only
+      - name: Native Build
+        run: |
+          mvn -V -B clean package -Dquarkus-core-only -DskipTests -DskipITs -pl 'app-metadata/deployment,app-metadata/runtime,common,http' -Dquarkus.profile=native -Dquarkus.native.container-runtime=docker
       - name: Zip Artifacts
         run: |
           zip -R artifacts-jvm${{ matrix.java }}.zip 'surefire-reports/*'
```

Reply:
On the second thought, I think we can proceed with this approach and optimize for resources later / if needed.
This gives us better view in PRs